test/WebJobs.Script.Tests.Integration/TestScripts/Proxies/proxies.json

```diff
@@ -178,6 +178,23 @@
       "requestOverrides": {
         "backend.request.headers.accept": "text/plain"
       }
+    },
+    "httpHead": {
+      "matchCondition": {
+        "route": "/headContentLengthTest",
+        "methods": [ "HEAD" ]
+      },
+      "backendUri": "https://httpbin.org/get"
+    },
+    "httpHeadToGet": {
+      "matchCondition": {
+        "route": "/headFromGetContentLengthTest",
+        "methods": [ "HEAD" ]
+      },
+      "backendUri": "https://httpbin.org/get",
+      "requestOverrides": {
+        "backend.request.method": "get"
```

Comment:
for this test, can you add the responseOverrides and change the response status to 304?